util/libcrypto.num

```diff
@@ -4699,17 +4699,17 @@ EVP_CIPHER_gettable_params              ?	3_0_0	EXIST::FUNCTION:
 EVP_CIPHER_settable_ctx_params          ?	3_0_0	EXIST::FUNCTION:
 EVP_CIPHER_gettable_ctx_params          ?	3_0_0	EXIST::FUNCTION:
 EVP_MD_get_params                       ?	3_0_0	EXIST::FUNCTION:
-EVP_MAC_fetch                           ?	3_0_0	EXIST::FUNCTION:
-EVP_MAC_settable_ctx_params             ?	3_0_0	EXIST::FUNCTION:
-EVP_MAC_set_ctx_params                  ?	3_0_0	EXIST::FUNCTION:
-EVP_MAC_get_ctx_params                  ?	3_0_0	EXIST::FUNCTION:
-EVP_MAC_gettable_ctx_params             ?	3_0_0	EXIST::FUNCTION:
-EVP_MAC_free                            ?	3_0_0	EXIST::FUNCTION:
-EVP_MAC_up_ref                          ?	3_0_0	EXIST::FUNCTION:
-EVP_MAC_get_params                      ?	3_0_0	EXIST::FUNCTION:
-EVP_MAC_gettable_params                 ?	3_0_0	EXIST::FUNCTION:
-EVP_MAC_provider                        ?	3_0_0	EXIST::FUNCTION:
-EVP_MAC_do_all_provided                 ?	3_0_0	EXIST::FUNCTION:
+EVP_MAC_fetch                           ?	3_0_0	NOEXIST::FUNCTION:
+EVP_MAC_settable_ctx_params             ?	3_0_0	NOEXIST::FUNCTION:
+EVP_MAC_set_ctx_params                  ?	3_0_0	NOEXIST::FUNCTION:
+EVP_MAC_get_ctx_params                  ?	3_0_0	NOEXIST::FUNCTION:
+EVP_MAC_gettable_ctx_params             ?	3_0_0	NOEXIST::FUNCTION:
+EVP_MAC_free                            ?	3_0_0	NOEXIST::FUNCTION:
+EVP_MAC_up_ref                          ?	3_0_0	NOEXIST::FUNCTION:
+EVP_MAC_get_params                      ?	3_0_0	NOEXIST::FUNCTION:
+EVP_MAC_gettable_params                 ?	3_0_0	NOEXIST::FUNCTION:
+EVP_MAC_provider                        ?	3_0_0	NOEXIST::FUNCTION:
+EVP_MAC_do_all_provided                 ?	3_0_0	NOEXIST::FUNCTION:
```

Comment:
And these.

Reply:
Resolved: sorry, slipped up there https://github.com/ckalina/openssl/commit/551eb1c3a14ab8b921c83dc85b3a38021227c403